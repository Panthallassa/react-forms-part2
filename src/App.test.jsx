import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("renders Todo App", () => {
	const { getByText } = render(<App />);
	expect(getByText(/Todo App/i)).toBeInTheDocument();
});

test("matches snapshot", () => {
	const { asFragment } = render(<App />);
	expect(asFragment()).toMatchSnapshot();
});
