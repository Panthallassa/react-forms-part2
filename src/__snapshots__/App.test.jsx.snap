// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    Todo App
  </h1>
  <form>
    <input
      placeholder="Enter new todo"
      type="text"
      value=""
    />
    <button
      type="submit"
    >
      Add Todo
    </button>
  </form>
</DocumentFragment>
`;
